### ISSUE TITLE
Error on fit

### ISSUE THREAD

**Issue by fernandohcardoso** [facebook/prophet#2227]
I've just installed prophet but It's not working when I try to fit.

```
14:11:57 - cmdstanpy - INFO - Chain [1] start processing
14:11:57 - cmdstanpy - INFO - Chain [1] done processing
14:11:57 - cmdstanpy - ERROR - Chain [1] error: terminated by signal 3221225657 
Optimization terminated abnormally. Falling back to Newton.
14:11:58 - cmdstanpy - INFO - Chain [1] start processing
14:11:58 - cmdstanpy - INFO - Chain [1] done processing
14:11:58 - cmdstanpy - ERROR - Chain [1] error: terminated by signal 3221225657 
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
~\anaconda3\lib\site-packages\prophet\models.py in fit(self, stan_init, stan_data, **kwargs)
    101         try:
--> 102             self.stan_fit = self.model.optimize(**args)
    103         except RuntimeError as e:

~\anaconda3\lib\site-packages\cmdstanpy\model.py in optimize(self, data, seed, inits, output_dir, sig_figs, save_profile, algorithm, init_alpha, tol_obj, tol_rel_obj, tol_grad, tol_rel_grad, tol_param, history_size, iter, save_iterations, require_converged, show_console, refresh, time_fmt)
    730             else:
--> 731                 raise RuntimeError(msg)
    732         mle = CmdStanMLE(runset)

RuntimeError: Error during optimization: 

During handling of the above exception, another exception occurred:

RuntimeError                              Traceback (most recent call last)
<ipython-input-12-ae8672abeabc> in <module>
      1 # Python
      2 m = Prophet()
----> 3 m.fit(df)

~\anaconda3\lib\site-packages\prophet\forecaster.py in fit(self, df, **kwargs)
   1169             self.params = self.stan_backend.sampling(stan_init, dat, self.mcmc_samples, **kwargs)
   1170         else:
-> 1171             self.params = self.stan_backend.fit(stan_init, dat, **kwargs)
   1172 
   1173         self.stan_fit = self.stan_backend.stan_fit

~\anaconda3\lib\site-packages\prophet\models.py in fit(self, stan_init, stan_data, **kwargs)
    108                 )
    109                 args['algorithm'] = 'Newton'
--> 110                 self.stan_fit = self.model.optimize(**args)
    111             else:
    112                 raise e

~\anaconda3\lib\site-packages\cmdstanpy\model.py in optimize(self, data, seed, inits, output_dir, sig_figs, save_profile, algorithm, init_alpha, tol_obj, tol_rel_obj, tol_grad, tol_rel_grad, tol_param, history_size, iter, save_iterations, require_converged, show_console, refresh, time_fmt)
    729                 get_logger().warning(msg)
    730             else:
--> 731                 raise RuntimeError(msg)
    732         mle = CmdStanMLE(runset)
    733         return mle

RuntimeError: Error during optimization: 
```

![image](https://user-images.githubusercontent.com/74622185/178040125-9891e163-5a23-4936-8ee1-baca13e54b45.png)

Could anyone help me?

**Comment by 7r1b0l1** [comment]
have the same problem, let me know if u fix it

**Comment by zangyu00544** [comment]
I encountered this problem too.

**Comment by Karlossal** [comment]
I have got mine working. 

I installed the cmdstanpy package and imported by using the following...

import cmdstanpy
cmdstanpy.install_cmdstan()
cmdstanpy.install_cmdstan(compiler=True)  # only valid on Windows

**Comment by fwong90** [comment]
I install a new dev enviroment and install all packages again. It works for me 100%

Im using anaconda, in the menu, chose enviroments and create new one, then select it.

**Comment by Akash-Jariwala** [comment]
> I install a new dev enviroment and install all packages again. It works for me 100%
> 
> Im using anaconda, in the menu, chose enviroments and create new one, then select it.

hey @fwong90 ,
I did same task which you listed out, it worked fine for me for the 1st time. But now after next day, when I am trying to run this, it again pops up with same same error.
After getting that same error, I tried removing an environment in which it was working. And tried same procedure by creating new environment. But now  it is not working for me.

Any other solution ???

**Comment by federicodecillia** [comment]
Same problem here.
I tried creating various new virtual environment, then typing `pip install prophet`

**Comment by sscoo** [comment]
Same problem

**Comment by BuckoIII** [comment]
I have the same problem. som additional info, i previously ran prophet in a conda environment and didn't have any issues. I recently changed from using conda envs to virtual envs and a brew python version and now it isn't working

**Comment by lzxTopC** [comment]
replace cmdstanpy version ,use pip install cmdstanpy==0.9.5 can resolve!!!!

**Comment by Karlossal** [comment]
Nope still didn’t resolve the issue

> replace cmdstanpy version ,use pip install cmdstanpy==0.9.5 can resolve!!!!

**Comment by sruap1214** [comment]
In my case was weird. It works by installing cmdstanpy==0.9.5. But i got the following error in the process

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
prophet 1.1.1 requires cmdstanpy>=1.0.4, but you have cmdstanpy 0.9.5 which is incompatible.

Despite teh preivous error, my code run

**Comment by lzxTopC** [comment]
yes,i known,but it can do work well,maybe is prophet is not mismatching cmdstanpy.

**Comment by oldrios** [comment]
> I have got mine working.
> 
> I installed the cmdstanpy package and imported by using the following...
> 
> import cmdstanpy cmdstanpy.install_cmdstan() cmdstanpy.install_cmdstan(compiler=True) # only valid on Windows

This works for me, thanks!

**Comment by krishnapunyakoti** [comment]
has anyone made this work on Mac M1? I am facing tough time making prophet work.

**Comment by sjfxking** [comment]
Keep only one Python and uninstall the other thoroughly

**Comment by sscoo** [comment]
> has anyone made this work on Mac M1? I am facing tough time making prophet work.

maybe try to run on Google Colab

**Comment by Shrikrishna11** [comment]
I got the same problem, then i installed "cmdstanpy.install_cmdstan(compiler=True)" in jupyter notebook and after getting output "true" it worked for me, so try to install 2-3 times if it is getting the output "False".definately it will work. then import cmdstanpy.

**Comment by cosmofan** [comment]
> replace cmdstanpy version ,use pip install cmdstanpy==0.9.5 can resolve!!!!

This works for me! thank you!

**Comment by Bigchicken98** [comment]
same problem, prophet==1.1.4，python=3.8，how to fix？

**Comment by aks43725** [comment]
> I have got mine working.
> 
> I installed the cmdstanpy package and imported by using the following...
> 
> import cmdstanpy cmdstanpy.install_cmdstan() cmdstanpy.install_cmdstan(compiler=True) # only valid on Windows

This works like a charm...

Python 3.10.11, prophet==1.1.4, cmdstanpy==1.1.0

**Comment by vivek-kumar007** [comment]
> I have got mine working.
> 
> I installed the cmdstanpy package and imported by using the following...
> 
> import cmdstanpy cmdstanpy.install_cmdstan() cmdstanpy.install_cmdstan(compiler=True) # only valid on Windows

This works but for every new session I need to run it again and it takes several minutes. How can I avoid running and installing it again and again?

**Comment by VenkyJishu** [comment]
having same problem for me as well and my setup is 
prophet is 1.1.1 and CmdStanPy is 1.1.0 and holidays is 0.18

error is :
RuntimeError: Error during optimization! Command 'C:\Anaconda\envs\prophet39_v2\Lib\site-packages\prophet\stan_model\prophet_model.bin random seed=81015 data file=C:\Users\Venky\AppData\Local\Temp\tmpd4k8ppj2\oyfodpq2.json init=C:\Users\Venky\AppData\Local\Temp\tmpd4k8ppj2\xhv3g1_o.json output file=C:\Users\Venky\AppData\Local\Temp\tmpd4k8ppj2\prophet_model1rdej18r\prophet_model-20241202115553.csv method=optimize algorithm=newton iter=10000' failed:

**Comment by valselt** [comment]
> I have got mine working.
> 
> I installed the cmdstanpy package and imported by using the following...
> 
> import cmdstanpy cmdstanpy.install_cmdstan() cmdstanpy.install_cmdstan(compiler=True) # only valid on Windows

THANK YOU SO MUCH! YOU'RE A LIVESAVER DUDE
